.env.example

```diff
@@ -41,6 +41,12 @@ EMAIL_SERVER_PORT=
 EMAIL_SERVER_USER=
 EMAIL_SERVER_PASSWORD=
 
+# GoCardless options
+NEXT_PUBLIC_GOCARDLESS_ENABLED=
+NEXT_PUBLIC_GOCARDLESS_COUNTRY=
```

Comment:
so it will work on one specific country alone?

Reply:
No, it should work world wide. The bank provider picker in the settings page fetch all bank providers. It works with either one country or undefined which then returns all the bank providers.
But because the list can be long in that case, I added the env to have a option to only fetch some.

If you have another approach to this, feel free to tell me.
One way is to have a country picker first and then the providers but it feels a bit weird..

Reply:
we have default currency option, we can get the country from that currency!

Reply:
I can't find the default currency? If you mean the currency that is set by the user when they try to add a expense it feels a bit weird to use that because it can change?

Reply:
So for now, it only works with one country in the package I use. https://github.com/nordigen/nordigen-node/blob/9d535ca9d788115d4e199c749432733be3ea6cdd/lib/api/institutions.js#L23

That can be fixed, either in the package or that we have a country picker first.
But if we think of the user case, will it be necessary to have multiple countries at the same instance?

Reply:
If we want to stick with the env and use it either with no value to fetch all bank providers or filter it based on the env. I have created a PR on the Nordigen-node package and hoping for a merge so we can use it.
https://github.com/nordigen/nordigen-node/pull/63